examples/samples/topicFilters.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  Namespace,
+  SubscriptionClient,
+  TopicClient,
+  SendableMessageInfo,
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage
+} from "../../lib";
+import * as dotenv from "dotenv";
+import { CorrelationFilter } from "../../lib/core/managementClient";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscriptionAllMessages = process.env.SUBSCRIPTION_ALL_MESSAGES || "";
+const subscriptionColorBlueSize10Orders = process.env.SUBSCRIPTION_COLOR_BLUE_ORDER_10 || "";
+const subscriptionColorRed = process.env.SUBSCRIPTION_CORRELATION_COLOR_RED || "";
+const subscriptionHighPriorityOrders = process.env.SUBSCRIPTION_HIGH_PRIORITY || "";
+
+console.log("str: ", str);
+console.log("path: ", topic);
+console.log("Subscription 1: ", subscriptionAllMessages);
+console.log("Subscription 2: ", subscriptionColorBlueSize10Orders);
+console.log("Subscription 3: ", subscriptionColorRed);
+console.log("Subscription 4: ", subscriptionHighPriorityOrders);
+
+let ns: Namespace;
+
+// This sample demonstrates how to use advanced filters with ServiceBus topics and subscriptions.
+// The sample creates a topic and 4 subscriptions with different filter definitions.
+// Each receiver will receive matching messages depending on the filter associated with a subscription.
+
+async function main(): Promise<void> {
+  var promises = new Array();
+  ns = Namespace.createFromConnectionString(str);
+
+  //Add different filter definitions in all the subscriptions.
+
+  //The first subscription selects all messages with a "true" filter, which is a SQL filter with an expression that is trivially true.
+  const subscriptionClientAllOrders = ns.createSubscriptionClient(topic, subscriptionAllMessages);
+  await removeAllRules(subscriptionClientAllOrders);
+  await addRules(subscriptionClientAllOrders, "AllOrders", "1=1");
+
+  //The second subscription selects all messages with a SQL filter with the user property 'color' having the value 'blue' and the 'quantity' user property having the value 10.
+  const subscriptionClientColorBlueSize10Orders = ns.createSubscriptionClient(
+    topic,
+    subscriptionColorBlueSize10Orders
+  );
+  await removeAllRules(subscriptionClientColorBlueSize10Orders);
+  await addRules(
+    subscriptionClientColorBlueSize10Orders,
+    "ColorBlueSize10Orders",
+    "color = 'blue' AND quantity = 10"
+  );
+
+  //The third subscription selects all messages with a SQL filter with the user property 'color' having the value 'red'.
```

Comment:
Do you have the linters enabled? I expect to get an error because there isn't a space between // and the start of the comment

Reply:
I just installed TSLint extension.I will change the code accordingly.